src/mlvis-common/ui/segment-panels/segment-group-panel.js

```diff
@@ -3,8 +3,7 @@ import PropTypes from 'prop-types';
 
 import SegmentButtonGroup from './segment-button-group';
 import {SegmentPanel} from '../styled-components';
-import {updateSegmentGroups} from '../../utils';
-import {COLORS} from '../../constants';
+import {updateSegmentGroups} from './utils';
```

Comment:
This action seems to be Manifold-specific, if it is not supposed to be used in other components/app, let's keep in Manifold instead of overwhelming `mlvis-common`. We need at least two (if not more) shared usages across components to justify the needs of moving it to the common section.